Library/Formula/bash.rb

```diff
@@ -6,16 +6,69 @@ class Bash < Formula
 
   stable do
     url "http://ftpmirror.gnu.org/bash/bash-4.3.tar.gz"
-    mirror "https://ftp.gnu.org/gnu/bash/bash-4.3.tar.gz"
```

Comment:
Could possibly set that up with a download strategy if you're particularly passionate about that. I'd like to avoid us having a bunch of mirrors for every GNU formula though because people never bother to check if they're actually up-to-date or still functional.